main/app.py

```diff
@@ -0,0 +1,231 @@
+from __future__ import print_function
+import random
+import sqlite3
+import sys
+import tkFileDialog
+from Tkinter import Tk
+
+from clint.textui import colored, puts
+from colorama import init, Back, Style  # Fore
+from pyfiglet import figlet_format
+from termcolor import cprint
+from .models.person import Person, Fellow, Staff
+from .models.room import Office, LivingSpace
+import ipdb
+
+
+class Amity(object):
+    """Description for amity"""
+
+    def __init__(self, db_name="amity.sqlite"):
+
+        self.conn = sqlite3.connect(db_name)
+        self.connect = self.conn.cursor()
+        self.rooms = {
+            'O': {}, 'L': {}
+        }
+        self.people = {
+            'staff and fellow': [],
+            'fellow to house': []
+        }
+        # People who have not been allocated to an office or living space
+        self.unallocated = {
+            'O': [], 'L': []
+        }
+
+        # People that have been fully occupied
+        self.allocated_office = {}
+        self.allocated_living = {}
+
+        self.space = {}
+
+    def create_rooms(self, rooms, room_type=None):
+        """Allows user to enter a list of room names specifying
+                whether office or living spaces"""
+        if room_type is None:
+
+            room_type = raw_input(
+                "Enter room type: \n O: Office space \n L: Living space: \n"
+            )
+            while room_type != "O" and room_type != "L":
+                room_type = raw_input(
+                    "Try again. Enter Room Type:\n O: Office space \n L: Living space: \n"
+                )
+        random.shuffle(rooms)
```

Comment:
Why shuffle the rooms here. Not sure how this is beneficial to us at the stage of room creation.